.github/workflows/tests.yml

```diff
@@ -0,0 +1,21 @@
+name: tests
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+    name: Build and Run Tests
+    runs-on: windows-latest
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v1
+      - name: Build and Run Tests
+        run: # execute your tests generating test results
```

Comment:
Am I wrong or this action does nothing? (there is no run for testing)

Reply:
Oops you are right

Reply:
What would I need to add?